-

```diff
@@ -0,0 +1,22 @@
+[package]
```

Comment:
what is this file?

Reply:
haha woops. I dunno how `-` got written.